api/C/include/csi_ll.h

```diff
@@ -36,5 +38,37 @@ typedef enum {
  */
 unsigned csi_ll_set_interrupt_enables(unsigned int_enables);
```

Comment:
Is this `unsigned` to be changed to `unsigned long` better, `unsigned long` size is as long as cpu register size, 4 bytes in 32bit, 8 bytes in 64bit.